package.json

```diff
@@ -86,5 +88,9 @@
     "tabWidth": 2,
     "semi": true,
     "singleQuote": true
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```

Comment:
Can you please apply the same rules as in the npm scripts. (I changed them just now a little bit)

```
"eslint": "eslint {src,cypress,demo}/**/*.js --fix ",
"prettier": "prettier --write {src,cypress,demo}/**/*.{js,css}",
```

Reply:
Done